digiroad2-oracle/src/main/resources/db/migration/V210__add_new_road_sign_side_column.sql

```diff
@@ -0,0 +1 @@
+alter table asset add sign_road_side char(1) default '0';
```

Comment:
Does it make sense to have this on the asset table instead of adding a new property?

Reply:
Make more sense in the properties, because it's not related to all assets, only for Traffic Signs